cts/scheduler/summary/promoted-with-blocked.summary

```diff
@@ -13,11 +13,11 @@ Current cluster status:
 
 Transition Summary:
   * Start      rsc1       (                   node2 )  due to unrunnable rsc3 start (blocked)
-  * Start      rsc2:0     (                   node3 )  due to colocation with rsc1 (blocked)
-  * Start      rsc2:1     (                   node4 )  due to colocation with rsc1 (blocked)
-  * Start      rsc2:2     (                   node5 )  due to colocation with rsc1 (blocked)
-  * Start      rsc2:3     (                   node1 )  due to colocation with rsc1 (blocked)
-  * Promote    rsc2:4     ( Stopped -> Promoted node2 )
+  * Start      rsc2:0     (                   node3 )
+  * Start      rsc2:1     (                   node4 )
+  * Start      rsc2:2     (                   node5 )
+  * Start      rsc2:3     (                   node1 )
+  * Promote    rsc2:4     ( Stopped -> Promoted node2 )  due to colocation with rsc1 (blocked)
```

Comment:
It doesn't feel ideal that the transition summary doesn't indicate that we're falling back to a start operation because the promote operation is blocked.

But I'm not immediately thinking of a good way to display and explain it, that doesn't take up too much horizontal space or look like there were two separate actions scheduled. The "Executing Cluster Transition" section shows it, at least. So this should be fine.

Reply:
Yeah it would be nice to improve that but not worth the effort right now. That's another thing that could use an overhaul, there are quite a few cases that could be worded better.